core/src/main/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinitions.scala

```diff
@@ -97,7 +96,7 @@ class SnowflakeTableDefinitions(conn: Connection) {
               val data = x.split(":")
               val name = data(0)
               val dataType = getDataType(data(1))
-              StructField(name, dataType, data(2).toBoolean)
+              StructField(name, dataType, data(2).toBoolean, null)
```

Comment:
```suggestion
              StructField(name, dataType, data(2).toBoolean, None)
```
don't use `null` for what can be `None`. it's a bug.

Reply:
done